starter-code/pizza.js

```diff
@@ -61,25 +71,98 @@ function renderGlutenFreeCrust() {
 
 function renderButtons() {
```

Comment:
So after some digging and consultation with the other TA's it was found that this can be done with a for loop, however the code is a bit fragile imo, as a change in button order or state inputs will cause everything to collapse:
Object.keys(state).forEach((item) => {
let btn = document.querySelector('.' + classMatchState[item]);
if (state[item]) {
btn.classList.add('active');
} else {
btn.classList.remove('active');
}
});

Kudos to Dalina I think.